svg-aam/META.yml

```diff
@@ -1,3 +1,3 @@
 spec: https://w3c.github.io/svg-aam/
 suggested_reviewers:
-  - halindrome
+  - cookiecrook
```

Comment:
Notes for Reviewers: 
1. Shane is no longer actively involved in this project, 
2. The to-be-deleted readme is related to the deprecated "ATTA" tests that were:
    a) never written for SVG-AAM
    b) require running manually, and
    c) not planned for SVG-AAM since we now have the ability to run these as automated tests.

Reply:
@cookiecrook Should there be a second reviewer for backup/greater coverage, and who can delegate if you're unavailable? 

(Someone important, not me 😛)

Reply:
Any reviewers can be assigned... This is just the auto-assignment feature. But since it sounds like you're volunteering, you're welcome to commit your time on it. 😉